dataactbroker/handlers/fileHandler.py

```diff
@@ -1506,13 +1506,29 @@ def add_list_submission_filters(query, filters):
         Raises:
             ResponseException - invalid type is provided for one of the filters
     """
+    # Checking for submission ID filter
     if 'submission_ids' in filters:
         sub_list = filters['submission_ids']
         if sub_list and isinstance(sub_list, list):
             sub_list = [int(sub_id) for sub_id in sub_list]
             query = query.filter(Submission.submission_id.in_(sub_list))
         elif sub_list:
             raise ResponseException("submission_ids filter must be null or an array", StatusCode.CLIENT_ERROR)
+    # Date range filter
+    if 'start_date' in filters or 'end_date' in filters:
```

Comment:
Shouldn't we also check that the `end_date` is later than the `start_date`?

Reply:
I don't think we do that with other date ranges on the backend. They give us a bad date range, they get no results. Do we want to alter how we do that here?

Reply:
I don't think that's within the scope of this ticket, but if we don't do it elsewhere we should. I can create a JIRA ticket.